.travis.yml

```diff
@@ -159,3 +164,8 @@ after_success:
       # Run script
       ./scripts/push_front.sh $TRAVIS_TAG
     fi
+
+  - |
+    if [[ "$ZDS_TEST_JOB" == *"doc"* ]]; then
```

Comment:
S’il te plaît, ajoute un commentaire `# upload built documentation to GitHub Pages` entre la ligne du `- |` et celle du `if [[…`. Ça fait plus joli dans les logs de Travis vu que la première ligne est affichée pour chaque section quand tout est replié :

![image](https://user-images.githubusercontent.com/15378830/29994882-39555ce8-8fdb-11e7-92bc-c2c5ca9bbe50.png)


:upside_down_face:

Reply:
J'approuve lourdement :p

Reply:
Done!